packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/javascript/guide.md

```diff
@@ -1,3 +1,406 @@
+    </head>
+    <body>
+      <div class="container">
+        <h1 class="text-center">Simple Web Page</h1>
+        <div id="messageBox" class="jumbotron">
+          You are not logged in.
+        </div>
+        <!-- where the sign-in form will be displayed -->
+        <div id="okta-login-container"></div>
+        <button id="logout" class="button" onclick="logout()" style="display: none">Logout</button>
+      </div>
+      <script type="text/javascript">
+        const oktaSignIn = new OktaSignIn({
+          baseUrl: "https://${yourOktaDomain}",
+          redirectUri: '{{https://${yourAppRedirectUri} configured in your OIDC app}}',
+          clientId: "${yourClientId}",
+          useInteractionCodeFlow: true,
+          pkce: true,
+          authParams: {
+            issuer: "https://${yourOktaDomain}/oauth2/default"
+          }
+        });
+
+        oktaSignIn.authClient.token.getUserInfo().then(function(user) {
+          document.getElementById("messageBox").innerHTML = "Hello, " + user.email + "! You are *still* logged in! :)";
+          document.getElementById("logout").style.display = 'block';
+        }, function(error) {
+          oktaSignIn.showSignInToGetTokens({
+            el: '#okta-login-container'
+          }).then(function(tokens) {
+            oktaSignIn.authClient.tokenManager.setTokens(tokens);
+            oktaSignIn.remove();
+
+            const idToken = tokens.idToken;
+            document.getElementById("messageBox").innerHTML = "Hello, " + idToken.claims.email + "! You just logged in! :)";
+            document.getElementById("logout").style.display = 'block';
+
+          }).catch(function(err) {
+            console.error(err);
+          });
+        });
+
+        function logout() {
+          oktaSignIn.authClient.signOut({ clearTokensAfterRedirect: true});
+          authClient.start();
+          location.reload();
+        }
+      </script>
+    </body>
+  </html>
+  ```
+
+3. Configure the code in `index.html` with values for your Okta org application integration:
+
+    * **baseUrl:** `"https://${yourOktaDomain}"`. For example, `"http://example.okta.com"`
+    * **redirectUri:** `"https://${yourAppRedirectUri}"`. For example, `"http://localhost:3000"`
+    * **clientId:** `"${yourClientId}"`. For example, `0oa2am3kk1CraJ8xO1d7`
+    * **issuer:** `"https://${yourOktaDomain}/oauth2/default"`. For example, `"https://example.okta.com/oauth2/default"`
+
+4. (Optional) Update the version of the `okta-auth-js` dependency to make use of other authentication features in your sample directory, `simple-spa`. See [Related SDKs](https://github.com/okta/okta-signin-widget#related-sdks). The basic authentication feature does not require this update.
+
+    ```bash
+    npm install @okta/okta-auth-js
+    ```
+
+    >**Note:** The SPA app logout function requires `okta-auth-js` 5.10 or greater.
```

Comment:
```suggestion
    >**Note:** The SPA app sign-out function requires `okta-auth-js` 5.10 or greater.
```

Reply:
The code snippet uses logout() as does the button in the UI. For those reasons, I'll keep it logout.